definitions/divorce/json/CaseEvent/CaseEvent-deemed-and-dispensed-nonprod.json

```diff
@@ -24,5 +24,18 @@
     "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/make-service-decision",
     "RetriesTimeoutURLAboutToSubmitEvent": "30,30",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "04/08/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "ServicePayment",
+    "Name": "Service payment exists",
+    "Description": "Service payment exists",
```

Comment:
Is this really the event's description?

Reply:
It wasnt specified in the story as to what the description was intended to be

Reply:
Fair. Can you please confirm with Jeremy where this is a clear enough name?

Reply:
Message sent

Reply:
No reply from Jeremy?